File: INF370.Group6.API/INF370.Group6.API/Layers/Core/Tenders/Tender.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;
using INF370.Group6.API.Layers.Core.Tenders;

namespace INF370.Group6.API.Layers.Core.Tenders
{
    public class Tender
    {
        public int Id { get; set; }
        [Required]
        [MaxLength(500)]
        public string Name { get; set; }
        [Required]
        [MaxLength(500)]
        public string Description { get; set; }
        public DateTime DateSubmitted { get; set; }

        public string TenderSource { get; set; }

        public int TenderStatusId { get; set; }
        public virtual TenderStatus Tenders { get; set; }
        public string TenderStatus { get; internal set; }
    }
}

File: INF370-Group6-Angular/src/app/@api/@api/tender/tender.ts
export interface Tender {
   TenderName: any;
   clientName: any;
   tenderStatus: any;
   TenderId: number,
   name: string,
   description: string,
   rawDateSubmitted: Date,
   dateSubmitted: string,
   tenderSource: string,
   tenderStatusId: number,
   tenderStatusName: string
}


export interface TenderStatus {
    id: number,
    name: string,
    description: string
    
}